src/com/sap/piper/Notify.groovy

```diff
@@ -0,0 +1,51 @@
+package com.sap.piper
+
+import groovy.text.SimpleTemplateEngine
+
+class Notify implements Serializable {
```

Comment:
What about `Logger`? Or to be more precise: having a dedicated logger class and a dedicated `Notify` class (if the later is really needed, the gap to `utils.pushToSWA()` is not that big and the parameters we use there are different each time anyway).